package/Dockerfile

```diff
@@ -0,0 +1,67 @@
+#
+# Builder image
+FROM registry.suse.com/bci/golang:1.22 AS builder
+
+ENV GOPATH=/go
+ENV PATH=$PATH:/usr/local/go/bin:$GOPATH/bin
+
+COPY config/ /src/config
+COPY server/ /src/server
+COPY vendor/ /src/vendor
+COPY go.mod go.sum adapter.go version.go Makefile /src
```

Comment:
This could be [optimised](https://github.com/rancher/cis-operator/blob/787cec3fc95d8f856bff5a446ebf06e122c6249d/package/Dockerfile#L38-L42) so that the Go mod download leverages Docker caching. Feel free to change that in that follow-up PR.